db/seeds.rb

```diff
@@ -279,7 +280,7 @@ def throw_into_past(res)
   res.due_date = res.due_date - factor
   mark_checked_out res
   mark_checked_in(res, res.equipment_model.category.max_checkout_length)
-  res.save(validate: false)
+  travel_to(res.start_date) { res.save }
```

Comment:
this marks overdue reservations as not overdue and probably messes with the status as well if the reservation is checked in